docs/pages/guides/tree-shaking.mdx

```diff
@@ -300,3 +302,211 @@ const { View, Image } = require('react-native-web');
 </Tab>
 
 </Tabs>
+
+## Unused export removal
+
+> Experimentally available in SDK 52 and greater
+
+In Expo SDK 52, you can experimentally enable support for automatically removing unused imports and exports across modules. This is useful for speeding up native OTA downloads and optimizing web performance where JavaScript must be parsed and executed using a standard JavaScript engine.
+
+Consider the following code:
+
+```js index.js
+import { ArrowUp } from './icons';
+
+export default function Home() {
+  return <ArrowUp />;
+}
+```
+
+```js icons.js
+export function ArrowUp() {
+  /* ... */
+}
+
+export function ArrowDown() {
+  /* ... */
+}
+
+export function ArrowRight() {
+  /* ... */
+}
+
+export function ArrowLeft() {
+  /* ... */
+}
+```
+
+Because only `ArrowUp` is used in `index.js`, the production bundle will remove all other components from `icons.js`.
+
+```js icons.js (Output)
+export function ArrowUp() {
+  /* ... */
+}
+```
+
+This system scales up to automatically optimize all import/export syntax in your app, across all platforms. While this results in smaller bundles, processing JS still requires time and computer memory so avoid importing millions of modules.
```

Comment:
```suggestion
This system scales up to automatically optimize all `import` and `export` syntax in your app, across all platforms. While this results in smaller bundles, processing JS still requires time and computer memory so avoid importing millions of modules.
```

Let's avoid using slash here between the import and export reference. Also, let's add the comma before